docs/admin/config/batch_changes.mdx

```diff
@@ -409,3 +430,38 @@ batchSpecLibrary(first: 100, labels: ["featured"]) {
   }
 }
 ```
+
+### Variables
+
+<Callout type="note">Variables are supported in Sourcegraph v6.6 and more.</Callout>
+
+Templates use variables to make batch specs adaptable to different scenarios. When creating a template, you can define placeholders for values that will vary between different uses of the template. Users filling out the template see these variables as form fields where they can enter specific values like repository names, file paths, or commit messages.
```

Comment:
"...make batch specs..." is a bit awkward to me ... maybe just say "...variables to customize batch specs."

Reply:
The use of "you" vs "user" here sets up a difference between the 2, but I don't think there is one, right? The "user" is the "you" here?

Reply:
The sentence `Templates use variables to customize batch specs.` doesn't transport the idea that you set it up once, and then many people reuse it, at least for me. I think I'm going to drop the entire sentence.